enhancements/node/lazy-image-pull.md

```diff
@@ -0,0 +1,266 @@
+#### Will any other components on the node change?
+
+Yes, configuration for the `c/storage` library used by CRI-O.
+
+## Operational Aspects of API Extensions
+
+#### For conversion/admission webhooks and aggregated apiservers: what are the SLIs (Service Level Indicators) an administrator or support can use to determine the health of the API extensions
+
+metrics `kubelet_pod_start_sli_duration_seconds`, `kubelet_pod_start_duration_seconds` abd `kubelet_runtime_operations_duration_seconds` can be used to gauge the performance of the container start up time.
+
+#### What impact do these API extensions have on existing SLIs (e.g. scalability, API throughput, API availability)
+
+Container that use large images but initiate the execution with a relatively small binary (e.g. using `ping` from fedora:39 container image) in that image should see faster start up time.
+
+#### How is the impact on existing SLIs to be measured and when (e.g. every release by QE, or automatically in CI) and by whom (e.g. perf team; name the responsible person and let them review this enhancement)
+
+Container start up time can be measured by an e2e CI test to validate the expected speed up.
+
+#### Describe the possible failure modes of the API extensions.
+
+MCO could encounter failures in rolling out the required machine config to enable this feature.
+
+#### Describe how a failure or behaviour of the extension will impact the overall cluster health (e.g. which kube-controller-manager functionality will stop working), especially regarding stability, availability, performance and security.
+
+If MCO fails to roll out required machine config to enable this feature, the container images that use `zstd:chunked` format will not observe improved start up times associated with their `zstd:chunked` image.
+
+#### Describe which OCP teams are likely to be called upon in case of escalation with one of the failure modes and add them as reviewers to this enhancement.
+
+OCP Node Team
+
+## Support Procedures
+
+### Detect the failure modes in a support situation, describe possible symptoms (events, metrics, alerts, which log output in which component)
+
+* If MCO fails to roll out required machine config to enable this feature, the container images that use zstd:chunked format will not observe any associated performance gains in their start up time.
+* If the workload is going to do heavy local disk IO, then this feature may contribute to higher resource memory and cpu usage on the node due to the use of FUSE filesystem.
+* Useful metrics to observe container start up time,  `kubelet_pod_start_sli_duration_seconds`, `kubelet_pod_start_duration_seconds` abd `kubelet_runtime_operations_duration_seconds`.
+
+### Disable the API extension (e.g. remove MutatingWebhookConfiguration xyz, remove APIService foo)
+
+#### What consequences does it have on the cluster health?
+
+Containers that use zstd:chunked image format will observe no associated performance gains in their start up time.
```

Comment:
See the “downgrade” section; just disabling the option will break the node, AFAICS.